nemo_curator/utils/distributed_utils.py

```diff
@@ -583,3 +585,29 @@ def performance_report_if(path=None, report_name="dask-profile.html"):
         return performance_report(os.path.join(path, report_name))
     else:
         return nullcontext()
+
+
+def seed_everything(seed: int = 42):
```

Comment:
Nit: We are probably going to call this a lot so I'd prefer something shorter like `seed_all`.

Reply:
Resolved by: https://github.com/NVIDIA/NeMo-Curator/pull/130/commits/f957b507d8d47e00cda0cf4bc64001e9f1546f4c